Mathlib/Order/ConditionallyCompleteLattice/Finset.lean

```diff
@@ -46,12 +46,173 @@ theorem Set.Finite.csSup_lt_iff (hs : s.Finite) (h : s.Nonempty) : sSup s < a 
 theorem Set.Finite.lt_csInf_iff (hs : s.Finite) (h : s.Nonempty) : a < sInf s ↔ ∀ x ∈ s, a < x :=
   @Set.Finite.csSup_lt_iff αᵒᵈ _ _ _ hs h
 
+theorem toDual_csSup {s : Set α} (h : s.Nonempty) (hs : Set.Finite s) :
+    OrderDual.toDual (sSup s) = sInf (OrderDual.toDual '' s) := by
```

Comment:
This is true for any set `s`, because this is basically `congr_arg sSup (image_id s)`

Reply:
Done, and made sure to generalize the existing lemmas out of `CompleteLattice` to `SupSet`.